examples/utils/utils.py

```diff
@@ -3,7 +3,7 @@
 import matplotlib.pyplot as plt
 import seaborn as sns
 import numpy as np
-from pandas import ewma
+import pandas as pd
```

Comment:
Why?

Reply:
Error in draw function because of the removal of ewma.